<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginWindow</class>
 <widget class="QWidget" name="loginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>459</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white</string>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameInput">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordInput">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>360</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="forgotPasswordButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>400</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Forgot Password?</string>
   </property>
  </widget>
  <widget class="QLabel" name="BackgroundPhoto">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>461</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../../../../../../../OneDrive - UBC/University of British Columbia/Courses/MANF_555 Factory Planning/Software Design Document/Stock Image/LoginPhoto.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>301</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white</string>
   </property>
   <property name="text">
    <string>Gateways MES</string>
   </property>
  </widget>
  <zorder>BackgroundPhoto</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>usernameInput</zorder>
  <zorder>passwordInput</zorder>
  <zorder>loginButton</zorder>
  <zorder>forgotPasswordButton</zorder>
  <zorder>label_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
